src/App.js

```diff
@@ -47,6 +47,7 @@ function App() {
           <option value="fr-FR">French - FR (Français)</option>
           <option value="pt-BR">pt-BR</option>
           <option value="tr-TR">Turkish (Türkçe)</option>
+          <option value="ar">Arabic - العربية</option>
```

Comment:
Are the parentheses not playing well with the locale, or why did you not include them?

Reply:
They're not playing well with RTL direction languages, I mentioned it here as well https://github.com/bndw/wifi-card/issues/80#issuecomment-887598899